controllers/apps/operations/switchover_util.go

```diff
@@ -467,7 +465,7 @@ func buildSwitchoverWorkloadEnvs(ctx context.Context,
 }
 
 // getServiceableNWritablePod returns the serviceable and writable pod of the component.
-func getServiceableNWritablePod(ctx context.Context, cli client.Client, cluster appsv1alpha1.Cluster, synthesizeComp component.SynthesizedComponent) (*corev1.Pod, error) {
+func getServiceableNWritablePod(ctx context.Context, cli client.Client, synthesizeComp component.SynthesizedComponent) (*corev1.Pod, error) {
```

Comment:
It is a weird name,  better to change name to 'getServingWritePod'

Reply:
N is short for And, so this name means "get Serviceable And Writable Pod", sounds reasonable